writing_checks/README.md

```diff
@@ -468,6 +468,38 @@ whereas if it run on a system with ALPS it will be translated to
 ddt --offline aprun ...
 ```
 
+## Custom Launchers
+
+ReFrame permits the simple addition of custom scheduler launchers. A launcher that invokes `mpirun` can for example be added as follows:
```

Comment:
I think the description (a single sentence could be enough) is better to be put here, since here is the point where (a) we elaborate a bit more on launchers and (b) the user here will need to know what a launcher really is.